src/Sylius/Bundle/ApiBundle/Resources/config/integrations/swagger.xml

```diff
@@ -18,53 +18,53 @@
     <services>
         <defaults public="false" />
 
-        <service id="Sylius\Bundle\ApiBundle\OpenApi\Factory\OpenApiFactory" decorates="api_platform.openapi.factory">
+        <service id="Sylius\Bundle\ApiBundle\OpenApi\Factory\OpenApiFactory" decorates="lexik_jwt_authentication.api_platform.openapi.factory">
```

Comment:
I'm not sure if it is needed

Reply:
If I see correctly, `lexik_jwt_authentication.api_platform.openapi.factory` decorates `api_platform.openapi.factory`, so this change seems to be ok